<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="result_show">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>421</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	
	color: rgb(0, 0, 0);
    
	
	background-color: rgb(154, 153, 150);
    border: 1px solid #ccc;
}
</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="seven">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	background-color: rgb(16, 10, 10);
	color: rgb(255, 255, 255);
	border: 1px solid #ccc
} </string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
    <property name="iconSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="equals">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>300</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	background-color: rgb(16, 10, 10);
	color: rgb(255, 255, 255);
	border: 1px solid #ccc
} </string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dot">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>300</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	background-color: rgb(16, 10, 10);
	color: rgb(255, 255, 255);
	border: 1px solid #ccc
} </string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="division">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>120</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	color: rgb(0, 0, 0);
	
	background-color: rgb(229, 165, 10);
	border: 1px solid #ccc
} </string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mul">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>180</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	color: rgb(0, 0, 0);
	
	background-color: rgb(229, 165, 10);
	border: 1px solid #ccc
} </string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sub">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>240</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>23</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	color: rgb(0, 0, 0);
	
	background-color: rgb(229, 165, 10);
	border: 1px solid #ccc
} </string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>300</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	color: rgb(0, 0, 0);
	
	background-color: rgb(229, 165, 10);
	border: 1px solid #ccc
} </string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mod">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>120</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	color: rgb(0, 0, 0);
	
	background-color: rgb(229, 165, 10);
	border: 1px solid #ccc
} </string>
    </property>
    <property name="text">
     <string>mod</string>
    </property>
   </widget>
   <widget class="QPushButton" name="degree">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>180</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	color: rgb(0, 0, 0);
	
	background-color: rgb(229, 165, 10);
	border: 1px solid #ccc
} </string>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="root">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>240</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	color: rgb(0, 0, 0);
	
	background-color: rgb(229, 165, 10);
	border: 1px solid #ccc
} </string>
    </property>
    <property name="text">
     <string>sqrt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sin">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>120</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	color: rgb(0, 0, 0);
	
	background-color: rgb(229, 165, 10);
	border: 1px solid #ccc
} </string>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ln">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>300</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	color: rgb(0, 0, 0);
	
	background-color: rgb(229, 165, 10);
	border: 1px solid #ccc
} </string>
    </property>
    <property name="text">
     <string>ln</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tan">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>240</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	color: rgb(0, 0, 0);
	
	background-color: rgb(229, 165, 10);
	border: 1px solid #ccc
} </string>
    </property>
    <property name="text">
     <string>tan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cos">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>180</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	color: rgb(0, 0, 0);
	
	background-color: rgb(229, 165, 10);
	border: 1px solid #ccc
} </string>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="log">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>300</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	color: rgb(0, 0, 0);
	
	background-color: rgb(229, 165, 10);
	border: 1px solid #ccc
} </string>
    </property>
    <property name="text">
     <string>log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="acos">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>180</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	color: rgb(0, 0, 0);
	
	background-color: rgb(229, 165, 10);
	border: 1px solid #ccc
} </string>
    </property>
    <property name="text">
     <string>acos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="atan">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>240</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	color: rgb(0, 0, 0);
	
	background-color: rgb(229, 165, 10);
	border: 1px solid #ccc
} </string>
    </property>
    <property name="text">
     <string>atan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="asin">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>120</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	color: rgb(0, 0, 0);
	
	background-color: rgb(229, 165, 10);
	border: 1px solid #ccc
} </string>
    </property>
    <property name="text">
     <string>asin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	color: rgb(0, 0, 0);
	
	background-color: rgb(229, 165, 10);
	border: 1px solid #ccc
} </string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="eight">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	background-color: rgb(16, 10, 10);
	color: rgb(255, 255, 255);
	border: 1px solid #ccc
} </string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
    <property name="iconSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="nine">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	background-color: rgb(16, 10, 10);
	color: rgb(255, 255, 255);
	border: 1px solid #ccc
} </string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
    <property name="iconSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="six">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>180</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	background-color: rgb(16, 10, 10);
	color: rgb(255, 255, 255);
	border: 1px solid #ccc
} </string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
    <property name="iconSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="four">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	background-color: rgb(16, 10, 10);
	color: rgb(255, 255, 255);
	border: 1px solid #ccc
} </string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
    <property name="iconSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="five">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>180</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	background-color: rgb(16, 10, 10);
	color: rgb(255, 255, 255);
	border: 1px solid #ccc
} </string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
    <property name="iconSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="one">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>240</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	background-color: rgb(16, 10, 10);
	color: rgb(255, 255, 255);
	border: 1px solid #ccc
} </string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="iconSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="two">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>240</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	background-color: rgb(16, 10, 10);
	color: rgb(255, 255, 255);
	border: 1px solid #ccc
} </string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
    <property name="iconSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="three">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>240</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	background-color: rgb(16, 10, 10);
	color: rgb(255, 255, 255);
	border: 1px solid #ccc
} </string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
    <property name="iconSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="zero">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>300</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	background-color: rgb(16, 10, 10);
	color: rgb(255, 255, 255);
	border: 1px solid #ccc
} </string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="iconSize">
     <size>
      <width>61</width>
      <height>61</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="bracket_1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	color: rgb(0, 0, 0);
	
	background-color: rgb(229, 165, 10);
	border: 1px solid #ccc
} </string>
    </property>
    <property name="text">
     <string>(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bracket_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	color: rgb(0, 0, 0);
	
	background-color: rgb(229, 165, 10);
	border: 1px solid #ccc
} </string>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="variable_x">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>300</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	color: rgb(0, 0, 0);
	
	background-color: rgb(229, 165, 10);
	border: 1px solid #ccc
} </string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>70</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="text">
     <string>x  =</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_x">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>70</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
